Formula/portable-openssl.rb

```diff
@@ -37,14 +42,18 @@ def arch_args
   end
 
   def configure_args
-    %W[
+    args = %W[
       --prefix=#{prefix}
       --openssldir=#{openssldir}
       no-ssl2
       zlib-dynamic
       no-shared
       enable-cms
     ]
+
+    args << "no-asm" if MacOS.version < :leopard
```

Comment:
Xcode 2.5's assembler is too old, and I'm not yet at the point of backporting a new assembler.